MediaPushPlugin/src/main/java/io/antmedia/plugin/MediaPushPlugin.java

```diff
@@ -92,7 +92,9 @@ public Map<String, RemoteWebDriver> getDrivers() {
 					"--no-sandbox",
 					"--autoplay-policy=no-user-gesture-required",
 					"--disable-background-media-suspend",
+					"--disable-gpu",
```

Comment:
Do we have any drawbacks for --disable-gpu?

I suspect that it may reduce the render rate and increase CPU usage which decreases the quality.

Reply:
Hi @mekya,

Yes, it will reduce the render rate but the `--disable-gpu` flag is only working with windows based systems so it won't effect on mac and linux. 

When we think about the windows, the only way of using the Ant Media Server is using wsl and it has small resource  so I think, this flag should be there.

Reply:
If it's only working for windows based system, there should not be an impact to us. Because AMS in running in Linux. 
Please correct me if I'm wrong.